src/components/AppSidebar/AppSidebar.vue

```diff
@@ -397,7 +424,7 @@ $desc-height: 46px;
 			position: relative;
 			padding: #{$desc-vertical-padding} #{$desc-menu-right-margin * 4} #{$desc-vertical-padding} $desc-vertical-padding / 2;
 			display: flex;
-			height: $desc-height / 2;
+			height: $desc-height * 0.5;
```

Comment:
Why? I feel like `/ 2` is more clear

Reply:
I changed it to `* 0.5`, because i also use `* 1.5` further down and i wanted it to be the same. I'm fine with changing it back though.